### PR TITLE
Croatia

### PR DESCRIPTION
**Pull request by PaulCornelissen**
- Parse exchanges from ENTSOE
- Configured zones.json to use ENTSOE parsers for consumption and consumptionForecast
- Wrote parser for HOPS json output according to issue #1950
- Add ENTSOE capacity figures to zones.json
- Took the liberty to add myself as contributor

### CODE REVIEW

File: config/zones.json [hunk 1]
```diff
@@ -2754,7 +2773,8 @@
       "wind": 4626
     },
     "contributors": [
-      "https://github.com/corradio"
+      "https://github.com/corradio",
```
Comment (corradio) [inline config/zones.json]: are you sure you didn't mean to update the key "HR"?

File: parsers/NL.py [hunk 2]
```diff
@@ -175,7 +175,7 @@ def get_production_data_energieopwek(date, session=None):
     # The API returns values per day from local time midnight until the last
     # round 10 minutes if the requested date is today or for the entire day if
     # it's in the past. 'sid' can be anything.
-    url = 'http://energieopwek.nl/jsonData.php?sid=2ecde3&Day=&Day=%s' % date.format('YYYY-MM-DD')
+    url = 'http://energieopwek.nl/jsonData.php?sid=2ecde3&Day=%s' % date.format('YYYY-MM-DD')
```
Comment (corradio) [inline parsers/NL.py]: 👍

File: config/zones.json (hunk 1 above)
Comment (PaulCornelissen) [reply config/zones.json]: I meant both if that's ok? What's the criteria?

Comment (corradio) [reply config/zones.json]: That's completely fine. But don't you also want to add yourself to CR as you update the CR production parser?

**Pull request merged**